Add the full OSU brand background palette and Beaver Orange text color to the
bootstrap_styles settings installed by osu_standard. The classes are generated
by madrone ($osu-bg-colors) and manzanita ($osu-bg-colors-cas in
components/_cas_backgrounds.scss); D7 larch backgrounds are mapped onto them
during migration by osu_migrations_cas CasLayoutBase.

--- a/config/install/bootstrap_styles.settings.yml
+++ b/config/install/bootstrap_styles.settings.yml
@@ -1,12 +1,12 @@
 layout_builder_theme: dark
-background_colors: "osu-bg-page-default|Default\r\nosu-bg-page-alt-1|White\r\nosu-bg-page-alt-2|Dark\r\nosu-bg-moondust|Moondust\r\nosu-bg-crater|Crater\r\nosu-bg-osuorange|Beaver Orange\r\nosu-bg-light-grey|Light Grey\r\n"
+background_colors: "osu-bg-page-default|Default\r\nosu-bg-page-alt-1|White\r\nosu-bg-page-alt-2|Dark\r\nosu-bg-moondust|Moondust\r\nosu-bg-crater|Crater\r\nosu-bg-osuorange|Beaver Orange\r\nosu-bg-light-grey|Light Grey\r\nosu-bg-black|Black\r\nosu-bg-pine-stand|Pine Stand\r\nosu-bg-high-tide|High Tide\r\nosu-bg-luminance|Luminance\r\nosu-bg-stratosphere|Stratosphere\r\nosu-bg-reindeer-moss|Reindeer Moss\r\nosu-bg-sea-foam|Sea Foam\r\nosu-bg-candela|Candela\r\nosu-bg-rogue-wave|Rogue Wave\r\nosu-bg-solar-flare|Solar Flare\r\nosu-bg-star-canvas|Star Canvas\r\nosu-bg-till|Till\r\nosu-bg-hop-bine|Hop Bine\r\nosu-bg-coastline|Coastline\r\nosu-bg-high-desert|High Desert\r\nosu-bg-trans-white|Semi-transparent White\r\nosu-bg-trans-black|Semi-transparent Black\r\n"
 background_image:
   bundle: image
   field: field_media_image
 background_local_video:
   bundle: local_video
   field: field_media_file
-text_colors: "osu-text-body|Default\r\nosu-text-bucktoothwhite|White"
+text_colors: "osu-text-body|Default\r\nosu-text-bucktoothwhite|White\r\nosu-text-osuorange|Beaver Orange"
 text_alignment: "bs-text-left|Left\r\nbs-text-center|Center\r\nbs-text-right|Right\r\nbs-text-justify|Justify"
 padding: "p-1|Padding 1\r\np-2|Padding 2\r\np-2-5|Padding 2.5\r\np-3|Padding 3\r\np-4|Padding 4\r\np-4-5|Padding 4_5\r\np-5|Padding 5\r\np-6|Padding 6\r\np-7|Padding 7"
 padding_left: "ps-1|Padding Left 1\r\nps-2|Padding Left 2\r\nps-2-5|Padding Left 2.5\r\nps-3|Padding Left 3\r\nps-4|Padding Left 4\r\nps-4-5|Padding Left 4_5\r\nps-5|Padding Left 5\r\nps-6|Padding Left 6\r\nps-7|Padding 7"
